From 45401b805059ab5cb8afbb07455c358783db39c7 Mon Sep 17 00:00:00 2001
From: David Bauer <blocktrron@users.noreply.github.com>
Date: Thu, 3 Oct 2019 01:44:20 +0200
Subject: [PATCH] ar71xx-generic: add ath10k packages to OCEDO Koala (#1838)

The OCEDO Koala was missing the correct package definition. Because of
this, firmware is potentially built with the wrong ath10k firmware /
driver.
---
 targets/ar71xx-generic | 1 +
 1 file changed, 1 insertion(+)

diff --git a/targets/ar71xx-generic b/targets/ar71xx-generic
index 1694cb0..d0b498f 100644
--- a/targets/ar71xx-generic
+++ b/targets/ar71xx-generic
@@ -133,6 +133,7 @@ fi
 # OCEDO
 
 device ocedo-koala koala
+packages $ATH10K_PACKAGES
 factory
 
 
-- 
2.7.4

